cogs/strike.py

```diff
@@ -705,9 +704,8 @@ async def on_member_update(self, before: discord.Member, after: discord.Member)
         async for audit_log_entry in main_guild.audit_logs(limit=5):
             FOUND_CORRECT_AUDIT_LOG_ENTRY: bool = (
                 audit_log_entry.target.id == after.id
-                and audit_log_entry.action == (
-                    discord.AuditLogAction.auto_moderation_user_communication_disabled
-                )
+                and audit_log_entry.action
+                == (discord.AuditLogAction.auto_moderation_user_communication_disabled)
```

Comment:
Comparison statements should not be split between lines. See above comment for further reasoning.